### PR TITLE
dev-db/unixODBC: fix incorrect ptr types with use -minimal

### PR DESCRIPTION
**Pull request by leakingmemory**
Bug: https://bugs.gentoo.org/936060
Closes: https://bugs.gentoo.org/936060

---

Please check all the boxes that apply:

- [ X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X ] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [ X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [ X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch [hunk 1]
```diff
@@ -0,0 +1,29 @@
+https://bugs.gentoo.org/936060
+--- a/Drivers/Postgre7.1/info.c
```
Comment (thesamesam) [inline dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch:2]: Could you email this to upstream please? (nick.gorham@easysoft.com)

I'll merge once that's done.

Comment (leakingmemory) [reply dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch:2]: Upstream commit https://github.com/lurcher/unixODBC/commit/45f501e1be2db6b017cc242c79bfb9de32b332a1

Comment (thesamesam) [reply dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch:2]: Ah, then it should've been in the patch description to begin with ;)

Could you add that to the top of the patch please? Thanks.

Comment (leakingmemory) [reply dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch:6]: I wasn't too familiar with what could be done in patch files. I've pushed a new description text now. Thanks for the feedback :)

Comment (thesamesam) [reply dev-db/unixODBC/files/unixODBC-2.3.12-bug-936060.patch:6]: Thank you! See https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions for future reference as well

**Comment by thesamesam** [comment]
Thanks!

**Pull request closed without merging**